src/getters.jl

```diff
@@ -1481,4 +1481,87 @@ function GetDrugAmounts(
 
 end
 
-export GetDatabasePersonIDs, GetPatientState, GetPatientGender, GetPatientRace, GetPatientAgeGroup, GetPatientVisits, GetMostRecentConditions, GetMostRecentVisit, GetVisitCondition, GetPatientEthnicity, GetDatabaseYearRange, GetVisitPlaceOfService, GetVisitConcept, GetVisitDate, GetDrugExposures, GetDrugConceptIDs, GetDrugAmounts
+"""
+GetCohortSubjects(cohort_ids, conn; tab = cohort_attribute)
```

Comment:
Hey @Jay-sanjay -- found the bug! This should not be the `cohort_attribute` table but rather the `cohort` table. You'll need to fix that everywhere within this PR where you reference a cohort. 

I am wondering if you had the fix on your machine and somehow missed updating the PR? Not entirely sure. But when this table is used instead, your test suite works perfectly.

Thanks!